### DISCUSSION TITLE
crossrefs-hover in HTML documentation

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-web#805] [General]
Hi,
in [HTML reference page](https://quarto.org/docs/reference/formats/html) currently `crossrefs-hover` is not documented.

Is it a bug or we do not have it, because it's a 1.4 and not 1.3 option?

Thank you

**Comment by jjallaire** [comment]
Correct, we don't show new features within the main documentation until after the formal release (so this isn't a bug)

**Reply by aborruso** [reply]
Thank you very much
